src/docker-osx-dev

```diff
@@ -1031,6 +1034,23 @@ function install {
 }
 
 #
+# Fetch the latest docker-osx-dev script, and then run install
+#
+function update_self {
+  log_info "Updating to the latest docker-osx-dev"
+  local readonly current_file=$(which docker-osx-dev)
```

Comment:
Hmm, should we use `which` or should it just use a bash-ism like `BASH_SOURCE[0]` to find itself?